var/spack/repos/builtin/packages/ascent/package.py

```diff
@@ -50,13 +50,13 @@ class Ascent(Package, CudaPackage):
     ###########################################################################
 
     variant("shared", default=True, description="Build Ascent as shared libs")
-    variant('test', default=True, description='Enable Ascent unit tests')
+    variant("test", default=True, description='Enable Ascent unit tests')
```

Comment:
for a follow-up: tests do not need to be a variant in Spack anymore.

They are controlled via `spack install --test=root ascent` which sets `self.run_tests`